scripts/test

```diff
@@ -7,4 +7,4 @@ echo Running tests
 
 PACKAGES=". $(find -name '*.go' | xargs -I{} dirname {} |  cut -f2 -d/ | sort -u | grep -Ev '(^\.$|.git|.trash-cache|vendor|bin)' | sed -e 's!^!./!' -e 's!$!/...!')"
 
-go test -race -cover -tags=test ${PACKAGES}
+go test -cover -tags=test ${PACKAGES}
```

Comment:
Can you only add -race if its amd64, I'd rather not globally drop it.  It is super useful.